ui/redux/reducers/blocked.js

```diff
@@ -33,7 +34,7 @@ export default handleActions(
     },
     [ACTIONS.USER_STATE_POPULATE]: (state: BlocklistState, action: { data: { blocked: ?Array<string> } }) => {
       const { blocked } = action.data;
-      const sanitizedBlocked = blocked && blocked.filter((e) => typeof e === 'string');
+      const sanitizedBlocked = [...new Set(blocked && blocked.filter((e) => typeof e === 'string'))];
       return {
         ...state,
         blockedChannels: sanitizedBlocked || state.blockedChannels,
```

Comment:
1. This changes the flow because now `sanitizedBlocked` will always an array even when `blocked` is null. The return statement wants to fall back to `state.blockedChannels` if `sanitizedBlocked` is null.
2. I recall `[...new Set` did not work on something and we removed them all.  Can't recall if it was Desktop only or old browsers ... the history seems squashed away. 
    - Then again, Anthony reintroduced 2 instances lately and we haven't heard complaints.  Then-then again, better to be safe and have a peace of mind.

Here's what we did in the past:
```
      if (!query) {
        // ensure no duplicates from double purchase bugs
        // return [...new Set(myPurchases)];
        return Array.from(new Set(myPurchases));
      }
```